### ISSUE TITLE
Error when installing Symfony 4.4-dev or when running bin/console after installation

### ISSUE THREAD

**Issue by jlslew** [symfony/symfony#32474]
**Symfony version(s) affected**: 4.4-dev

**Description**  

```
Executing script cache:clear [KO]
 [KO]
Script cache:clear returned with error code 255
!!  PHP Fatal error:  Uncaught Symfony\Component\ErrorCatcher\Exception\ClassNotFoundException: Attempted to load class "ErrorHandler" from namespace "Symfony\Component\Debug".
!!  Did you forget a "use" statement for e.g. "Symfony\Component\ErrorCatcher\ErrorHandler", "PhpParser\ErrorHandler" or "Monolog\ErrorHandler"? in ~/test/vendor/symfony/framework-bundle/FrameworkBundle.php:71
!!  Stack trace:
!!  #0 ~/test/vendor/symfony/http-kernel/Kernel.php(137): Symfony\Bundle\FrameworkBundle\FrameworkBundle->boot()
!!  #1 ~/test/vendor/symfony/framework-bundle/Console/Application.php(169): Symfony\Component\HttpKernel\Kernel->boot()
!!  #2 ~/test/vendor/symfony/framework-bundle/Console/Application.php(75): Symfony\Bundle\FrameworkBundle\Console\Application->registerCommands()
!!  #3 ~/test/vendor/symfony/console/Application.php(150): Symfony\Bundle\FrameworkBundle\Console\Application->doRun(Objec in ~/test/vendor/symfony/framework-bundle/FrameworkBundle.php on line 71
!!  
Script @auto-scripts was called via post-update-cmd
```

**How to reproduce**  

- `composer create-project symfony/website-skeleton test -s dev`
- `bin/console`

**Additional context**  

<details>
 <summary>Output</summary>
Installing symfony/website-skeleton (dev-master 1c8ef260e32a14b890a00116daaf2dc9116eae57)

  - Installing symfony/website-skeleton (dev-master 1c8ef26): Cloning 1c8ef260e3 from cache

Created project in ../test

Loading composer repositories with package information

Updating dependencies (including require-dev)

Package operations: 1 install, 0 updates, 0 removals

  - Installing symfony/flex (dev-master 9c13d9a): Loading from cache

Symfony operations: 1 recipe (ca4ff8119ea41edd7b7b204263cf02c0)

  - Configuring symfony/flex (>=1.0): From github.com/symfony/recipes:master

Loading composer repositories with package information

Updating dependencies (including require-dev)

Package operations: 103 installs, 0 updates, 0 removals

  - Installing symfony/flex (dev-master 9c13d9a): Loading from cache

  - Installing ocramius/package-versions (1.4.0): Loading from cache

  - Installing psr/log (dev-master c4421fc): Loading from cache

  - Installing symfony/polyfill-php73 (dev-master 639cc79): Loading from cache

  - Installing symfony/polyfill-mbstring (dev-master fe5e94c): Loading from cache

  - Installing symfony/polyfill-intl-idn (dev-master eff4216): Loading from cache

  - Installing symfony/mime (dev-master fe974bf): Loading from cache

  - Installing symfony/event-dispatcher-contracts (dev-master c61766f): Loading from cache

  - Installing psr/container (dev-master 014d250): Loading from cache

  - Installing psr/cache (dev-master 78c5a01): Loading from cache

  - Installing symfony/event-dispatcher (4.4.x-dev 030b51a): Loading from cache

  - Installing symfony/error-catcher (dev-master 4b8a354): Loading from cache

  - Installing symfony/debug (dev-master 089cae8): Loading from cache

  - Installing symfony/http-foundation (dev-master bb90fb6): Loading from cache

  - Installing symfony/http-kernel (4.4.x-dev 030b536): Loading from cache

  - Installing symfony/service-contracts (dev-master f391a00): Loading from cache

  - Installing symfony/dependency-injection (4.4.x-dev b524c1e): Loading from cache

  - Installing symfony/routing (4.4.x-dev 9614790): Loading from cache

  - Installing symfony/finder (4.4.x-dev 9353a97): Loading from cache

  - Installing symfony/filesystem (4.4.x-dev 4e3b24f): Loading from cache

  - Installing symfony/config (4.4.x-dev 8f5bef5): Loading from cache

  - Installing symfony/cache-contracts (dev-master ec5524b): Loading from cache

  - Installing symfony/var-exporter (dev-master 31fa55c): Loading from cache

  - Installing symfony/cache (dev-master f57bdf1): Loading from cache

  - Installing symfony/framework-bundle (4.4.x-dev 2a6aaa7): Loading from cache

  - Installing doctrine/lexer (dev-master 975c0e4): Loading from cache

  - Installing doctrine/annotations (1.7.x-dev 3f35255): Loading from cache

  - Installing doctrine/reflection (dev-master 4a97a22): Loading from cache

  - Installing doctrine/event-manager (dev-master e05ae70): Loading from cache

  - Installing doctrine/collections (1.6.x-dev c5e0bc1): Loading from cache

  - Installing doctrine/cache (dev-master 17ec59c): Loading from cache

  - Installing doctrine/persistence (1.2.x-dev 43526ae): Loading from cache

  - Installing sensio/framework-extra-bundle (dev-master d30e4ab): Loading from cache

  - Installing symfony/asset (dev-master 6b5e1d6): Loading from cache

  - Installing symfony/dotenv (dev-master 04a9bd9): Loading from cache

  - Installing symfony/expression-language (dev-master dbf3457): Loading from cache

  - Installing symfony/inflector (dev-master 1069771): Loading from cache

  - Installing symfony/property-access (dev-master 30fe2ef): Loading from cache

  - Installing symfony/options-resolver (dev-master 803d221): Loading from cache

  - Installing symfony/polyfill-intl-icu (dev-master c324f62): Loading from cache

  - Installing symfony/intl (dev-master 96c5750): Loading from cache

  - Installing symfony/form (dev-master d07fe0a): Loading from cache

  - Installing symfony/http-client-contracts (dev-master 970bca2): Loading from cache

  - Installing symfony/http-client (dev-master 331189f): Loading from cache

  - Installing monolog/monolog (1.x-dev 17cbfb8): Loading from cache

  - Installing symfony/monolog-bridge (4.4.x-dev e6162e1): Loading from cache

  - Installing symfony/monolog-bundle (dev-master 7fbecb3): Loading from cache

  - Installing symfony/stopwatch (4.4.x-dev 5c288f8): Loading from cache

  - Installing symfony/console (4.4.x-dev d39c11c): Loading from cache

  - Installing zendframework/zend-code (dev-develop 8003f74): Loading from cache

  - Installing ocramius/proxy-manager (2.2.x-dev 41cac6e): Loading from cache

  - Installing doctrine/dbal (dev-master ea92d56): Loading from cache

  - Installing doctrine/inflector (1.3.x-dev 45d9b13): Loading from cache

  - Installing doctrine/common (dev-master 3773c7b): Loading from cache

  - Installing doctrine/migrations (dev-master 0069f40): Loading from cache

  - Installing symfony/security-core (dev-master f3366d1): Loading from cache

  - Installing symfony/doctrine-bridge (4.4.x-dev 4c7a6b8): Loading from cache

  - Installing doctrine/doctrine-cache-bundle (dev-master c46725e): Loading from cache

  - Installing jdorn/sql-formatter (dev-master 7ef9b85): Loading from cache

  - Installing doctrine/doctrine-bundle (1.12.x-dev ead3f06): Loading from cache

  - Installing symfony/yaml (4.3.x-dev c60ecf5): Loading from cache

  - Installing doctrine/doctrine-migrations-bundle (dev-master 2fa7e62): Loading from cache

  - Installing doctrine/instantiator (dev-master 7c71fc2): Loading from cache

  - Installing doctrine/orm (2.7.x-dev 9e73672): Loading from cache

  - Installing symfony/orm-pack (dev-master 36c2a92): Loading from cache

  - Installing symfony/process (dev-master 308af67): Loading from cache

  - Installing symfony/security-http (dev-master c8b290d): Loading from cache

  - Installing symfony/security-guard (dev-master b5306cc): Loading from cache

  - Installing symfony/security-csrf (dev-master 1b7504d): Loading from cache

  - Installing symfony/security-bundle (dev-master 0480317): Loading from cache

  - Installing symfony/translation-contracts (dev-master cb4b18a): Loading from cache

  - Installing symfony/translation (4.4.x-dev 425fecb): Loading from cache

  - Installing symfony/serializer (4.4.x-dev e0c3efd): Loading from cache

  - Installing symfony/property-info (dev-master c3eef9e): Loading from cache

  - Installing webmozart/assert (1.4.0): Loading from cache

  - Installing phpdocumentor/reflection-common (1.0.1): Loading from cache

  - Installing phpdocumentor/type-resolver (0.4.0): Loading from cache

  - Installing phpdocumentor/reflection-docblock (4.3.1): Loading from cache

  - Installing symfony/serializer-pack (dev-master c5f18ba): Loading from cache

  - Installing egulias/email-validator (2.1.9): Loading from cache

  - Installing swiftmailer/swiftmailer (dev-master a53dab2): Loading from cache

  - Installing symfony/swiftmailer-bundle (dev-master cb125b3): Loading from cache

  - Installing symfony/validator (4.4.x-dev 001dafd): Loading from cache

  - Installing psr/link (dev-master 131b5c3): Loading from cache

  - Installing fig/link-util (dev-master 1a07821): Loading from cache

  - Installing symfony/web-link (dev-master 83239fc): Loading from cache

  - Installing symfony/var-dumper (dev-master 2352074): Loading from cache

  - Installing twig/twig (2.x-dev e74d477): Loading from cache

  - Installing symfony/twig-bridge (dev-master 89946c9): Loading from cache

  - Installing symfony/twig-bundle (4.4.x-dev f867468): Loading from cache

  - Installing symfony/web-profiler-bundle (dev-master 0592404): Loading from cache

  - Installing symfony/profiler-pack (dev-master 99c4370): Loading from cache

  - Installing symfony/debug-bundle (dev-master 4cb56f2): Loading from cache

  - Installing easycorp/easy-log-handler (v1.0.7): Loading from cache

  - Installing symfony/debug-pack (dev-master 09a4a1e): Loading from cache

  - Installing nikic/php-parser (dev-master 3f718ee): Loading from cache

  - Installing symfony/maker-bundle (v1.12.0): Loading from cache

  - Installing symfony/css-selector (dev-master ba2e533): Loading from cache

  - Installing symfony/dom-crawler (dev-master a492541): Loading from cache

  - Installing symfony/browser-kit (dev-master f78146a): Loading from cache

  - Installing symfony/phpunit-bridge (dev-master bbccbc7): Loading from cache

  - Installing symfony/test-pack (dev-master ff87e80): Loading from cache

  - Installing symfony/web-server-bundle (dev-master 02ec27c): Loading from cache

Writing lock file

Generating autoload files

Symfony operations: 20 recipes (8c3102f252d3a61dfb09bfce666c091e)

  - Configuring symfony/framework-bundle (>=4.2): From github.com/symfony/recipes:master

  - Configuring symfony/routing (>=4.2): From github.com/symfony/recipes:master

  - Configuring doctrine/annotations (>=1.0): From github.com/symfony/recipes:master

  - Configuring sensio/framework-extra-bundle (>=5.2): From github.com/symfony/recipes:master

  - Configuring symfony/monolog-bundle (>=3.3): From github.com/symfony/recipes:master

  - Configuring symfony/console (>=3.3): From github.com/symfony/recipes:master

  - Configuring doctrine/doctrine-cache-bundle (>=dev-master): From auto-generated recipe

  - Configuring doctrine/doctrine-bundle (>=1.6): From github.com/symfony/recipes:master

  - Configuring doctrine/doctrine-migrations-bundle (>=1.2): From github.com/symfony/recipes:master

  - Configuring symfony/security-bundle (>=3.3): From github.com/symfony/recipes:master

  - Configuring symfony/translation (>=3.3): From github.com/symfony/recipes:master

  - Configuring symfony/swiftmailer-bundle (>=2.5): From github.com/symfony/recipes:master

  - Configuring symfony/validator (>=4.3): From github.com/symfony/recipes:master

  - Configuring symfony/twig-bundle (>=3.3): From github.com/symfony/recipes:master

  - Configuring symfony/web-profiler-bundle (>=3.3): From github.com/symfony/recipes:master

  - Configuring symfony/debug-bundle (>=4.1): From github.com/symfony/recipes:master

  - Configuring easycorp/easy-log-handler (>=1.0): From github.com/symfony/recipes:master

  - Configuring symfony/maker-bundle (>=1.0): From github.com/symfony/recipes:master

  - Configuring symfony/phpunit-bridge (>=4.3): From github.com/symfony/recipes:master

  - Configuring symfony/web-server-bundle (>=3.3): From github.com/symfony/recipes:master

ocramius/package-versions:  Generating version class...

ocramius/package-versions: ...done generating version class

Executing script cache:clear [KO]

 [KO]

Script cache:clear returned with error code 255

!!  PHP Fatal error:  Uncaught Symfony\Component\ErrorCatcher\Exception\ClassNotFoundException: Attempted to load class "ErrorHandler" from namespace "Symfony\Component\Debug".

!!  Did you forget a "use" statement for e.g. "Symfony\Component\ErrorCatcher\ErrorHandler", "PhpParser\ErrorHandler" or "Monolog\ErrorHandler"? in ~/test/vendor/symfony/framework-bundle/FrameworkBundle.php:71

!!  Stack trace:

!!  #0 ~/test/vendor/symfony/http-kernel/Kernel.php(137): Symfony\Bundle\FrameworkBundle\FrameworkBundle->boot()

!!  #1 ~/test/vendor/symfony/framework-bundle/Console/Application.php(169): Symfony\Component\HttpKernel\Kernel->boot()

!!  #2 ~/test/vendor/symfony/framework-bundle/Console/Application.php(75): Symfony\Bundle\FrameworkBundle\Console\Application->registerCommands()

!!  #3 ~/test/vendor/symfony/console/Application.php(150): Symfony\Bundle\FrameworkBundle\Console\Application->doRun(Objec in ~/test/vendor/symfony/framework-bundle/FrameworkBundle.php on line 71

!!  
Script @auto-scripts was called via post-update-cmd
</details>

**Comment by javiereguiluz** [comment]
The problem is that you are using Symfony 4.4, which hasn't been released yet (it will release on November 2019) and it's making lots of changes these days related to Debug, exceptions and the new components introduced about this. Hopefully this will be fixed in a few days. Thanks.

**Comment by jlslew** [comment]
Closing, fixed by commit eda49e295ec5353319774a7695f8c3dfa5c95cfb
